yarn.lock

```diff
@@ -2985,7 +2985,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@mui/utils@npm:^6.1.6":
+"@mui/utils@npm:^6.1.6, @mui/utils@npm:^6.4.3":
```

Comment:
@jonkafton I still think this is good to merge, as the Yarn install still works fine, but why was this changed?